rust/src/quic/error.rs

```diff
@@ -24,6 +24,7 @@ pub enum QuicError {
     StreamTagNoMatch(u32),
     InvalidPacket,
     Incomplete,
+    UnhandledYet,
```

Comment:
NotSupported?

Reply:
Ok.
Is it clear that it means not supported _by Suricata_ ?